### PR TITLE
feat: post and patch user risk profile

### PR DESCRIPTION
**Pull request by marilia-calado**
This route both posts and patches the user risk profile and populates risk's table.

### CODE REVIEW

File: app/controllers/user_risk_controller.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    user_id = decoded_jwt.get("sub")
+    user: UserModel = UserModel.query.get(user_id)
+
+    if not user:
+        raise UserNotFound
+
+    selected_case = ""
+
+    for case_key, case_value in RiskModel.CASES.items():
+        # TODO: find a better name to count variable
```
Comment (marcelosj3) [inline app/controllers/user_risk_controller.py:28]: I can't think of anything good to name this variable either.

File: app/models/risk_model.py [hunk 2]
```diff
@@ -18,5 +17,34 @@ class RiskModel(db.Model):
...
     text = Column(String, nullable=False)
+
+    CASES = {
+        "NOT_SPECIFIED": {"live_nearby_river": False, "live_nearby_mountain": False},
+        "EARTH_SLIDING": {"live_nearby_river": False, "live_nearby_mountain": True},
+        "FLOOD": {"live_nearby_river": True, "live_nearby_mountain": False},
+        "FLOOD_AND_SLIDING": {"live_nearby_river": True, "live_nearby_mountain": True},
+    }
+
+    # TODO: find a better way to display the texts
```
Comment (marcelosj3) [inline app/models/risk_model.py:29]: Maybe we should get rid of the triple strings and just display it as a long string in here, i think the triple strings creates a few "\n" inside the string and might not be what we want, but for now we can keep it like this, idk.

File: app/controllers/user_risk_controller.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+from sqlalchemy.orm import Query, Session
+
+from app.configs.database import db
+from app.exceptions.user_exc import UserNotFound
+from app.models.risk_model import RiskModel
+from app.models.user_model import UserModel
+
+
+@jwt_required()
+def create_user_risk_profile():
```
Comment (marcelosj3) [inline app/controllers/user_risk_controller.py:14]: Please create a task to componentize the functions into services, there is no need to to that now, but later in the project a few of these functions might be useful to reuse.

**Review by marcelosj3 — APPROVED** [review]

File: app/models/risk_model.py (hunk 2 above)
Comment (marilia-calado) [reply app/models/risk_model.py:29]: I'll do it, then.

File: app/models/risk_model.py [hunk 4]
```diff
@@ -18,5 +17,34 @@ class RiskModel(db.Model):
...
+    VALUES = {
+        "NOT_SPECIFIED": """Atenção! A sua região apresenta previsão para chuvas fortes (ou violentas <depende da milimetragem>).
```
Comment (penguinuux) [inline app/models/risk_model.py]: Messages must contain a maximum of 160 characters (SMS limit). We need to change it in the future.

File: app/models/risk_model.py [hunk 5]
```diff
@@ -18,5 +17,34 @@ class RiskModel(db.Model):
...
+        "FLOOD_AND_SLIDING": {"live_nearby_river": True, "live_nearby_mountain": True},
+    }
+
+    # TODO: find a better way to display the texts
+    VALUES = {
+        "NOT_SPECIFIED": """Atenção! A sua região apresenta previsão para chuvas fortes (ou violentas <depende da milimetragem>). 
+                   
+                   Se você mora em região de encostas ou próximo a um rio ou córrego, reúna seus pertences mais importantes 
+                   (documentos, dinheiro ou cartões) e se encaminhe para o ponto de apoio da Defesa Civil mais próximo.""",
+        "EARTH_SLIDING": """Atenção! A sua região apresenta previsão para chuvas fortes (ou violentas <depende da milimetragem>).
```
Comment (penguinuux) [inline app/models/risk_model.py]: Messages must contain a maximum of 160 characters (SMS limitation).

**Review by penguinuux — CHANGES_REQUESTED** [review]

**Review by penguinuux — APPROVED** [review]

**Pull request merged**